includes/components/class-vendor.php

```diff
@@ -49,6 +49,9 @@ public function __construct( $args = [] ) {
 			// Alter templates.
 			add_filter( 'hivepress/v1/templates/listing_view_page', [ $this, 'alter_listing_view_page' ] );
 			add_filter( 'hivepress/v1/templates/user_edit_settings_page/blocks', [ $this, 'alter_user_edit_settings_page' ], 100, 2 );
+		} else {
+
+			add_action( 'hivepress/v1/models/vendor/update', [ $this, 'update_user' ], 100, 2 );
```

Comment:
There's no guarantee that this is a vendor editor update, vendor can be updated when the order changes, or via other unpredictable actions (this hook is fired even if some vendor meta is updated). It may be better to check only the editor update, maybe via save_post or another way.